### DISCUSSION TITLE
Disk usage from log files

### DISCUSSION THREAD

**Discussion by AlexGeek77** [Athou/commafeed#1192] [General]
I noticed my free space was quickly disappearing on my Commafeed container. When I looked in the log folder I saw this. Is there a way to set the logging level and maybe offer compressing the older log files automatically? 
```
root@commafeed:/opt/commafeed/log# ls -alh 
total 13G
drwxr-xr-x 2 root root 4.0K Jan 26 00:00 .
drwxr-xr-x 3 root root 4.0K Jan 26 10:33 ..
-rw-r--r-- 1 root root 104M Jan 21 23:59 commafeed-2024-01-21.log
-rw-r--r-- 1 root root 4.1G Jan 22 23:59 commafeed-2024-01-22.log
-rw-r--r-- 1 root root 1.3G Jan 24 00:00 commafeed-2024-01-23.log
-rw-r--r-- 1 root root 1.2G Jan 24 23:59 commafeed-2024-01-24.log
-rw-r--r-- 1 root root 4.5G Jan 25 23:59 commafeed-2024-01-25.log
-rw-r--r-- 1 root root 2.0G Jan 26 10:33 commafeed.log
```
I'm running my Commafeed server in a Proxmox LXC with Debian 12.

**Comment by Athou** [comment]
> Is there a way to set the logging level

Yes, have a look in your `config.yml` file. However, CommaFeed should not log that much by default, your log files are huge. What is the content of those log files, do you have a an error that keeps occuring or something?

**Comment by AlexGeek77** [comment]
Here's a [Pastebin link](https://pastebin.com/zULCVfZ8) to the last 100 lines of my most recent log. I've updated the `config.yml` file to go back to INFO level logging. I think I upped it to DEBUG when I was working through the Fever API connection.
